en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -207,16 +207,101 @@ For more information on initializing an API Project using OpenAPI/Swagger Specif
 The Automation Server can be configured to run a specific pipeline for promoting artifacts to other environments. 
 The DevOps team can develop this pipeline further to include automated tests, workflow approvals, and other tasks.  
 
-The **apictl** tool should be installed in the automation servers to begin the process. As the tool supports a variety of platforms, including Linux/Windows and macOS, this can be done easily. The **apictl** tool supports the process of importing API Projects via the `import-api` command. This command is mainly used in the pipeline to migrate to different environments.
+The **apictl** tool should be installed in the automation servers to begin the process. As the tool supports a variety of platforms, including Linux/Windows and macOS, this can be done easily. 
 
-1.  Import the `SwaggerPetstore` API into the production environment and test the API by running the following sample command.
+#### a. Promoting API project(s) in a GIT repository using a unified approach
+
+The repository (Version Control System) needs to be cloned into the instance that is executing the CI/CD process. From 3.2.0, **apictl** has the inbuilt support to integrate with a GIT based version control system. It gives a unified command `vcs deploy` to deploy any type of project (APIs, API Products and Apps).
```

Comment:
```suggestion
The repository (Version Control System) needs to be cloned into the instance that is executing the CI/CD process. From 3.2.0 onwards, **apictl** has the inbuilt support to integrate with a GIT based version control system. It gives a unified command `vcs deploy` to deploy any type of project (APIs, API Products and Apps).
```

Reply:
This repository is created by the user. In one of the previous steps - https://github.com/wso2/docs-apim/blame/master/en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md#L162. So it's difficult to provide an exact link. I will reword it to indicate the above. Hope that would be ok?